core/src/main/java/nl/nn/adapterframework/align/XmlTypeToJsonSchemaConverter.java

```diff
@@ -111,6 +114,13 @@ public XSElementDeclaration findElementDeclaration(String elementName, String na
 	public JsonStructure createJsonSchema(String elementName, XSElementDeclaration elementDecl) {
 		JsonObjectBuilder builder = Json.createObjectBuilder();
 		definitionsBuilder = Json.createObjectBuilder();
+		builder.add("$schema", JSON_SCHEMA);
+		if(elementDecl.getNamespace() != null){
+			builder.add("$id", elementDecl.getNamespace());
```

Comment:
is het id de namespace? Het moet een unieke identifier zijn..

Reply:
Zie https://json-schema.org/understanding-json-schema/basics.html#id4